packages/esm-messages-app/src/messages/messages-chart-dashboard.component.tsx

```diff
@@ -0,0 +1,195 @@
+            >
+              {t('add', 'Add')}
+            </Button>
+          </div>
+        </CardHeader>
+        <DataTable
+          rows={tableRows}
+          headers={headers}
+          isSortable
+          size={isTablet ? 'lg' : 'sm'}
+          useZebraStyles
+          overflowMenuOnHover={isDesktop}
+        >
+          {({ rows, headers, getHeaderProps, getTableProps, getRowProps }) => (
+            <>
+              <TableContainer>
+                <Table aria-label="messages summary" {...getTableProps()}>
+                  <TableHead>
+                    <TableRow>
+                      {headers.map((header) => (
+                        <TableHeader
+                          className={classNames(styles.productiveHeading01, styles.text02)}
+                          {...getHeaderProps({
+                            header,
+                            isSortable: header.isSortable,
+                          })}
+                        >
+                          {header.header?.content ?? header.header}
+                        </TableHeader>
+                      ))}
+                      <TableHeader />
+                    </TableRow>
+                  </TableHead>
+                  <TableBody>
+                    {rows.map((row) => (
+                      <TableRow key={row.id} {...getRowProps({ row })}>
+                        {row.cells.map((cell) => (
+                          <TableCell key={cell.id}>{cell.value?.content ?? cell.value}</TableCell>
+                        ))}
+                        <TableCell className="cds--table-column-menu">
+                          <Layer className={styles.layer}>
+                            <OverflowMenu
+                              aria-label="Edit message type"
+                              size={isTablet ? 'lg' : 'sm'}
+                              flipped
+                              align="left"
+                            >
+                              <OverflowMenuItem
+                                className={styles.menuItem}
+                                id="editMessageType"
+                                onClick={() => launchEditMessagesForm(row.id)}
+                                itemText={t('edit', 'Edit')}
+                              />
+                            </OverflowMenu>
+                          </Layer>
+                        </TableCell>
+                      </TableRow>
+                    ))}
+                  </TableBody>
+                </Table>
+              </TableContainer>
+
+              {rows.length === 0 ? (
+                <div className={styles.tileContainer}>
+                  <Tile className={styles.tile}>
+                    <div className={styles.tileContent}>
+                      <p className={styles.content}>{t('noConditionsToDisplay', 'No messages to display')}</p>
+                      <p className={styles.helper}>{t('checkFilters', 'Check the filters above')}</p>
```

Comment:
Is there any filtering here at all? Otherwise, why mention it. It might be better to, e.g., put a button to add a message or something here if there are none? That's a pattern we also use.